includes/Modules/Sign_In_With_Google/Compatibility_Checks/Compatibility_Check.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Base abstract class for Sign In With Google compatibility checks.
+ *
+ * @package   Google\Site_Kit\Modules\Sign_In_With_Google
+ * @copyright 2025 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
```

Comment:
We usually include class name at the top of the class docbloc 
```suggestion
/**
 * Class Google\Site_Kit\Modules\Sign_In_With_Google\Compatibility_Checks\Compatibility_Check
 *
 * @package   Google\Site_Kit\Modules\Sign_In_With_Google
 * @copyright 2025 Google LLC
 * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
 * @link      https://sitekit.withgoogle.com
 */
```

Reply:
This makes no sense. The first line should usually be used to explain the purpose of the class instead of repeating the class name itself. I think we should either not write anything there or something useful.

Reply:
Updated